.github/actions/trivy/action.yml

```diff
@@ -39,28 +39,34 @@ runs:
       id: pull
       shell: bash
       run: |
+        mkdir -p $GITHUB_WORKSPACE/.cache/trivy/db
         oras pull ghcr.io/temporalio/trivy-db:2
+        tar -xzf db.tar.gz -C $GITHUB_WORKSPACE/.cache/trivy/db
+        rm db.tar.gz
+
+        mkdir -p $GITHUB_WORKSPACE/.cache/trivy/java-db
         oras pull ghcr.io/temporalio/trivy-java-db:1
+        tar -xzf javadb.tar.gz -C $GITHUB_WORKSPACE/.cache/trivy/java-db
+        rm javadb.tar.gz
+
+    - name: Install Trivy
+      uses: aquasecurity/setup-trivy@v0.2.3
+      with:
+        version: v0.65.0
 
     - name: Scan Container Image
       id: scan
-      uses: aquasecurity/trivy-action@0.28.0
+      shell: bash
       env:
-        TRIVY_DB_REPOSITORY: ghcr.io/temporalio/private-actions/trivy-db,public.ecr.aws/aquasecurity/trivy-db,ghcr.io/aquasecurity/trivy-db
-        TRIVY_JAVA_DB_REPOSITORY: ghcr.io/temporalio/private-actions/trivy-java-db,public.ecr.aws/aquasecurity/trivy-java-db,ghcr.io/aquasecurity/trivy-java-db
-        # Not 100% sure if these are required, but so far no issues.
-        TRIVY_USERNAME: ${{ github.actor }}
-        TRIVY_PASSWORD: ${{ github.token }}
-      with:
-        cache: true
-        exit-code: 0
-        image-ref: ${{ steps.vars.outputs.tag }}
-        format: 'sarif'
-        output: 'trivy-${{ steps.vars.outputs.name }}-results.sarif'
+        TRIVY_SKIP_DB_UPDATE: true
+        TRIVY_SKIP_JAVA_DB_UPDATE: true
+        TRIVY_CACHE_DIR: ${{ github.workspace }}/.cache/trivy
+      run: |
+        trivy image --severity HIGH,CRITICAL --no-progress ${{ steps.vars.outputs.tag }} --format sarif --output trivy-${{ steps.vars.outputs.name }}-results.sarif
 
-    - name: Upload ${{ inputs.image-name }} image Trivy scan results to GitHub Security tab
+    - name: Upload Trivy scan results
       uses: github/codeql-action/upload-sarif@v3
-      #if: always()
+      if: always()
```

Comment:
Intentional?

Reply:
This is! Required because we removed the exit code flag: https://github.com/temporalio/docker-builds/pull/285#discussion_r2248490363